src/components/MapView/DateSelector/index.test.tsx

```diff
@@ -5,6 +5,8 @@ import DateSelector from '.';
 
 import { store } from '../../../context/store';
 
+jest.mock('../../Notifier', () => 'mock-Notifier');
```

Comment:
Shouldn't we write this test in `index.test.tsx` in `Notifer` folder instead of here in the `DateSelctor` tests

Reply:
This is actually mocking the Notifier to NOT test it in here.